pkg/v1/nodegroup/requests_opts.go

```diff
@@ -53,6 +53,9 @@ type CreateOpts struct {
 
 	// AutoscaleMaxNodes represents maximum possible number of worker nodes in the nodegroup.
 	AutoscaleMaxNodes *int `json:"autoscale_max_nodes,omitempty"`
+
+	// UserData represents base64 data which is used to pass a shell script that worker nodes run on boot.
+	UserData *string `json:"user_data,omitempty"`
```

Comment:
Do we need a pointer here? Seems not, could you give more context in case we actually need it

Reply:
In MKS API user data logic was already realized as a pointer
Using pointer here has no negative effect, but a little bit superfluously, I agree